crates/burn-jit/src/tests/mod.rs

```diff
@@ -111,28 +123,37 @@ macro_rules! testgen_jit {
         burn_tensor::testgen_calibration!();
         burn_tensor::testgen_scheme!();
         burn_tensor::testgen_quantize!();
-    };
+    }
 }
 
 #[macro_export]
 macro_rules! testgen_jit_fusion {
     () => {
+        $crate::testgen_jit_fusion!(f32: [], i32: []);
+    };
+    ($f_def:ident: [$($float:ident),*], $i_def:ident: [$($int:ident),*]) => {
         use super::*;
         use burn_jit::tests::{burn_autodiff, burn_fusion, burn_ndarray, burn_tensor};
 
         pub type TestBackend = burn_fusion::Fusion<JitBackend<TestRuntime, f32, i32>>;
+        pub type TestBackend2<F, I> = burn_fusion::Fusion<JitBackend<TestRuntime, F, I>>;
```

Comment:
TestBackend2 is the one that can be injected with the precision right? Not sure if we could come up with a better name.

Reply:
I couldn't come up with a better name and it's only used in the macro to instantiate and override `TestBackend` with the correct types, so I left it for now. It's not used in the actual tests, just the macro.